src/Features/VisualBasic/Portable/Completion/CompletionProviders/AwaitCompletionProvider.vb

```diff
@@ -0,0 +1,65 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Collections.Immutable
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.Completion
+Imports Microsoft.CodeAnalysis.Completion.Providers
+Imports Microsoft.CodeAnalysis.Host.Mef
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
+    <ExportCompletionProvider(NameOf(AwaitCompletionProvider), LanguageNames.VisualBasic)>
+    <ExtensionOrder(After:=NameOf(KeywordCompletionProvider))>
+    <[Shared]>
+    Friend NotInheritable Class AwaitCompletionProvider
+        Inherits AbstractAwaitCompletionProvider
+
+        <ImportingConstructor>
+        <Obsolete(MefConstruction.ImportingConstructorMessage, True)>
+        Public Sub New()
+        End Sub
+
+        Public Overrides ReadOnly Property TriggerCharacters As ImmutableHashSet(Of Char) = CommonTriggerChars
+
+        Private Protected Overrides ReadOnly Property AsyncKeywordTextWithSpace As String = "Async "
```

Comment:
i don't know why, but this bugs me.  can jyst you get hte asynckeyword, and hten add the space in the root? :)